src/index.js

```diff
@@ -34,7 +34,7 @@ if (module.hot) {
       <AppContainer>
         <NextApp store={store} history={syncedHistory} />
       </AppContainer>
-      ), rootElement
+      ), rootElement,
```

Comment:
I can't understant this.
I tried `eslint --fix`, but I don't see any different.